src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutes.scala

```diff
@@ -45,19 +45,19 @@ abstract class LeoRoutes(val leonardoService: LeonardoService, val proxyService:
               }
             }
           }
-        }
```

Comment:
Just fixes an indentation mistake in https://github.com/broadinstitute/leonardo/pull/54: I meant for `requireUserInfo` to encompass everything under this path.